libexec/cli/exec.exec

```diff
@@ -83,5 +83,10 @@ SINGULARITY_IMAGE="${1:-}"
 export SINGULARITY_IMAGE
 shift
 
+if [ -d "$SINGULARITY_IMAGE" ]; then
+  export SINGULARITY_USERNS=1
```

Comment:
Ahh, you are re-exec'ing the SUID version if necessary from within sexec... Considering this can not be the default code pathway, I'm hesitant to make it default even if the image is a directory...